_limitations/set_limitations.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Unsupported SET Statements 
+nav_order: 2
+---
+
+## Unsupported SET statements
+
+The following SET statements are available for use with Babelfish versions 1.2.0 and higher:
```

Comment:
Technically SET @variable = value is also a SET statement and that is of course supported

Reply:
Soon, this will only list Unsupported features - we changed direction midtable, and started documenting the functionality we added/supported instead of just unsupported features.  I'm still weeding through that.

Reply:
This will be addressed in a subsequent PR to avoid merge conflicts...